pantheon/src/main/java/tech/pegasys/pantheon/cli/options/MetricsCLIOptions.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.pantheon.cli.options;
+
+import tech.pegasys.pantheon.metrics.prometheus.MetricsConfiguration;
+
+import java.util.Arrays;
+import java.util.List;
+
+import picocli.CommandLine;
+
+public class MetricsCLIOptions implements CLIOptions<MetricsConfiguration.Builder> {
```

Comment:
Not for this PR, but long term do you see us migrating the metrics HTTP Server CLI options here too?

Reply:
Yeah, I think it would be great if we could start grouping all of the related options regardless of whether they're experimental or not.